.github/workflows/workflow-generate-ci-documentation.yml

```diff
@@ -0,0 +1,64 @@
+name: Generate CI Documentation
+on:
+  workflow_call:
+    inputs:
+      commit_resources:
+        description: 'Commit the resources after generating them. (Requires the access_token to be passed)'
+        required: false
+        default: false
+        type: boolean
+    secrets:
+      access_token:
+        description: 'The access token to use for commits.'
+        required: false
+  workflow_dispatch:
+    branches:
+    - main
+    - develop
+    - "feature-*"
+    - "release-*"
+    inputs:
+      commit_resources:
+        description: 'commit the resources after generating them. Requires a PAT defined as secrets.COMMIT_TOKEN'
+        required: true
+        default: false
+        type: boolean
+jobs:
+  update_ci_documentation:
+    runs-on: ubuntu-20.04
+    steps:
+      - name: Generate CI Documentation
+        uses: Compton-NIST/Inspector@v1
+        with:
+          artifact_name: docs_ci
+          output_path: docs_ci
+          file_prefix: OSCAL
+      - name: Make Overview the README.md for the directory.
+        run: |
+          cp docs_ci/OSCAL.workflows.md docs_ci/README.md
+          rm -f docs_ci/OSCAL.workflows.md
```

Comment:
Blocking: do we need to parameterize these accordingly, such that changes on L33 and L44 in the current instance of this PR, if they change, would not have to be duplicated here?

If that is complex for now, we can defer the name for another time. If you have limited bandwidth, just adjust the the updated paths to `build/ci-cd` and do not worry about parameterization. Does that make sense?

Reply:
Yeah, I missed that.  If I feel productive in the morning, I might add an input for the output markdown so it can be defined through the action, rather than have this step.  If I'm rushing, I'll adjust this step for now.